README.md

```diff
@@ -1,3 +1,3 @@
 # SchoolWebsite
-test
-K.H
\ No newline at end of file
+
+## This is a school website that is being made by 4 Mount Sion pupils with the help of RedHat 
```

Comment:
I would suggest to remove the `##` as this setting the text as header when we want this to be a paragraph of regular text, also I would suggest to rewording this paragraph also